pkg/sql/pg_extension.go

```diff
@@ -44,9 +106,7 @@ CREATE TABLE pg_extension.geography_columns (
 	srid integer,
 	type text
 )`,
-	generator: func(ctx context.Context, p *planner, db *DatabaseDescriptor) (virtualTableGenerator, cleanupFunc, error) {
-		return nil, func() {}, errors.Newf("not yet implemented")
-	},
+	populate: postgisColumnsTablePopulator(types.GeographyFamily),
```

Comment:
Are there any permissions that we need to consider here?

Reply:
i added one fix